### PR TITLE
Add tokens for content notification new design

### PR DESCRIPTION
**Pull request by Rhotimee**
#### Summary
Implement new design for content notification

### CODE REVIEW

**Comment by CarlosCortizasCT** [comment]
It seems text color within the notification should be `color-solid` in the new theme but currently it uses a color based on the notification's type.

UX design:
image


Deployment preview:
image

**Comment by Rhotimee** [comment]
> It seems text color within the notification should be `color-solid` in the new theme but currently it uses a color based on the notification's type.
> 
> UX design: image
> 
> Deployment preview: image

The Figma design is saying otherwise. @FilPob, which is using the correct color?

File: packages/components/notifications/src/content-notification/content-notification.tsx [hunk 1]
```diff
@@ -112,13 +144,43 @@ NotificationIcon.displayName = 'NotificationIcon';
...
+    case 'warning':
+      return designTokens.borderColorForContentNotificationWhenWarning;
+    case 'success':
+      return designTokens.borderColorForContentNotificationWhenSuccess;
+    default:
+      return '';
+  }
+};
+
+const getContainerBackgroundColour = (props: TContentNotificationProps) => {
```
[the remark this answers is not in the document]
Comment (Rhotimee) [reply packages/components/notifications/src/content-notification/content-notification.tsx]: Updated [51301d9](https://github.com/commercetools/ui-kit/pull/2396/commits/51301d95f9727133bd04ae8c2114c5e7f3b1e88f). Thanks

**Review by tdeekens — APPROVED** [review]

**Comment by Rhotimee** [comment]
> It seems text color within the notification should be `color-solid` in the new theme but currently it uses a color based on the notification's type.

I confirmed from @FilPob and you're right. Thanks. 
[f7232d0](https://github.com/commercetools/ui-kit/pull/2396/commits/f7232d0a385abec16821eb5d7a88aa50e847872d)

**Review by CarlosCortizasCT — APPROVED** [review]
Cool 😎

**Review by FilPob — APPROVED** [review]

**Pull request merged**